checker/jtreg/nullness/NonRawTypeArgumentTest.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * @test
+ * @summary Test case for type argument in method invocation.
```

Comment:
I find this summary confusing and don't see how this matches the code - there is no type argument there.
You need to say something about what type argument you mean.
Same with the test name 'NonRawTypeArgument` isn't very helpful - all type arguments are non-raw, otherwise there wouldn't be type arguments...